### PR TITLE
update(pr-template) Request reviewer to confirm documentation update/…

### PR DESCRIPTION
**Pull request by EugeneHlushko**
…issue

**What kind of change does this PR introduce?**
Update PR template

**Did you add tests for your changes?**
N/A

**If relevant, link to documentation update:**
N/A

**Summary**
Make it obvious that documentation update or filing a linked issue under webpack/webpack.js.org is a required step before merging a new feature or functionality change

**Does this PR introduce a breaking change?**
No

**Other information**

preview:

- [ ] I confirm that authors had documented their feature or submitted an issue to [webpack.js.org](https://github.com/webpack/webpack.js.org) linking to this PR with a feature documentation request only  **if applicable**.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -22,3 +23,8 @@
 <!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->
 
 **Other information**
+
+<!-- Leave this untouched, for reviewers of your PR. -->
+**Checklist for reviewers**
+
+- [ ] I confirm that authors had documented their feature or submitted an issue to webpack/webpack.js.org linking to this PR with a feature documentation request if applicable.
```
Comment (ooflorent) [inline .github/PULL_REQUEST_TEMPLATE.md]: Could you add a link to repository?

**Comment by EugeneHlushko** [comment]
@ooflorent though it will format from webpack/webpack.js.org but it didnt 👎 
updated, thx

**Comment by EugeneHlushko** [comment]
Good point regarding issue template, will add it too

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -8,9 +8,10 @@
 
 <!-- Note that we won't merge your changes if you don't add tests -->
 
-**If relevant, link to documentation update:**
+**If relevant, link to documentation update or issue:**
 
 <!-- Link PR from webpack/webpack.js.org here, or N/A -->
+<!-- Note: If you are implementing a feature or functionality change make sure that you have updated the documentation or submited an issue linking to this PR under webpack/webpack.js.org repository requesting documentation update -->
```
Comment (montogeek) [inline .github/PULL_REQUEST_TEMPLATE.md]: Use a link, people are lazy.

Comment (EugeneHlushko) [reply .github/PULL_REQUEST_TEMPLATE.md]: yup, good idea

**Comment by sokra** [comment]
I'm a bit skeptical about that. Does it really make sense to document a feature before the PR has merged? Maybe it's never merged, stuff is renamed or it's merged in another way. But if you feel comfortable with it, I'll happily accept the current PR.

Here is another proposal (could also be added in addition to this PR):

For each merged PR we let the @webpack-bot copy the PR description as issue into webpack/webpack.js.org with some heading `Document that:` and mention the PR author. It would create a bit of spam but you could easily close all irrelevant issues. The benefit is you'll never miss a PR.

**Comment by TheLarkInn** [comment]
@sokra I think that's a great idea. In a perfect world I'd really love to have them merge together but I understand how hard that is. 

In addition to having the information copied over, we should have the bot collect the issue it was proposed (if applicable), the PR that merged (for verbosity and easy for new contributors to reference the work/changes made).

**Comment by EugeneHlushko** [comment]
Guys idea is not to merge them together, idea is that it is required to at least notify documentation team that you had merged a new feature in an issue.

**Comment by EugeneHlushko** [comment]
The bot action looks very nice 👍

**Pull request closed without merging**